cpu/cpu_windows.go

```diff
@@ -19,16 +21,26 @@ var cpuMap = map[string]string{
 	"machdep.cpu.stepping":     "stepping",
 }
```

Comment:
Unrelated to your changes but let's fix it anyway: `cpuMap` is not used on Windows, could you remove the variable declaration?